apps/indexer-coordinator/src/project/project.model.ts

```diff
@@ -160,6 +267,10 @@ export class ProjectEntity {
   @Field()
   queryEndpoint: string; // endpoint of query service
 
+  @Column('jsonb', { default: {} })
+  @Field()
+  serviceEndpoints: IProjectNetworkEndpoints | IProjectRpcEndpoints;
```

Comment:
The keys of endpoints are decided by project type + chainFamily when project is rpc.
```suggestion
  serviceEndpoints: Record<string, string>;
```